h/static/styles/partials-v2/_search.scss

```diff
@@ -90,20 +107,6 @@
   flex-grow: 1;
 }
 
-.search-result-bucket__annotations-count-container {
```

Comment:
_Please_ resist the urge to make changes that aren't related to your PR! I just spent two minutes trying to work out how this change was related to the intent of the PR, before realising that you've just reordered some CSS.

There's nothing wrong with reordering and tidying, but _please please please_ put it in a separate PR.